CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # v0.16.0-SNAPSHOT
 * Upgraded dependencies:
     * argonaut-6.2
+	* jetty-9.4.4.v20170414
+	* tomcat-8.5.14
```

Comment:
Tabs?!  GH markdown processor kept the indents all one level deep, so not a big deal.